ido-vertical-mode.el

```diff
@@ -87,7 +87,7 @@ so we can restore it when turning `ido-vertical-mode' off")
 (defvar ido-vertical-count-active nil
   "Used internally to track whether we're already showing the count")
 
-(defcustom ido-vertical-define-keys 'C-n-and-C-p-only
```

Comment:
hmm I'm reluctant to change this default from `C-n-and-C-p-only` since that means that all users must explicitly do a `(setq ido-vertical-define-keys...` on their own. 

Is it enough to remove the `M-p` binding for `ido-toggle-prefix` below and update the readme ? Then, users who explicitly want `ido-toggle-prefix` somewhere can move it on their own, and everyone still gets `C-n/C-p` out of the box, since it makes sense for the vertical list ?